exercises/minesweeper/test.ml

```diff
@@ -11,121 +11,120 @@ let format_board strings =
+      let expected = ["1*2*1"] in
+      ae expected (annotate b)
+    );
   "horizontal line, mines at edges" >:: (fun _ ->
-    let b = ["*   *"] in
-    let expected = ["*1 1*"] in
-    ae expected (annotate b)
-  );
+      let b = ["*   *"] in
+      let expected = ["*1 1*"] in
+      ae expected (annotate b)
+    );
   "vertical line" >:: (fun _ ->
-    let b = [" ";
-             "*";
-             " ";
-             "*";
-             " "] in
-    let expected = ["1";
-                    "*";
-                    "2";
-                    "*";
-                    "1"] in
-    ae expected (annotate b)
-  );
+      let b = [" ";
+               "*";
+               " ";
+               "*";
+               " "] in
+      let expected = ["1";
+                      "*";
+                      "2";
+                      "*";
+                      "1"] in
+      ae expected (annotate b)
+    );
   "vertical line, mines at edges" >:: (fun _ ->
-    let b = ["*";
-             " ";
-             " ";
-             " ";
-             "*"] in
-    let expected = ["*";
-                    "1";
-                    " ";
-                    "1";
-                    "*"] in
-    ae expected (annotate b)
-  );
+      let b = ["*";
+               " ";
+               " ";
+               " ";
+               "*"] in
+      let expected = ["*";
+                      "1";
+                      " ";
+                      "1";
+                      "*"] in
+      ae expected (annotate b)
+    );
   "cross" >:: (fun _ ->
-    let b = ["  *  ";
-             "  *  ";
-             "*****";
-             "  *  ";
-             "  *  "] in
-    let expected = [" 2*2 ";
-                    "25*52";
-                    "*****";
-                    "25*52";
-                    " 2*2 "] in
-    ae expected (annotate b)
-  );
-  "large board" >:: (fun _ ->
-    let b = [" *  * ";
-             "  *   ";
-             "    * ";
-             "   * *";
-             " *  * ";
-             "      "] in
-    let expected = ["1*22*1";
-                    "12*322";
-                    " 123*2";
-                    "112*4*";
-                    "1*22*2";
-                    "111111"] in
-    ae expected (annotate b)
-  );
+      let b = ["  *  ";
+               "  *  ";
+               "*****";
+               "  *  ";
+               "  *  "] in
+      let expected = [" 2*2 ";
+                      "25*52";
+                      "*****";
+                      "25*52";
+                      " 2*2 "] in
+      ae expected (annotate b)
+    );
+  "large minefield" >:: (fun _ ->
+      let b = [" *  * ";
+               "  *   ";
+               "    * ";
+               "   * *";
+               " *  * ";
+               "      "] in
+      let expected = ["1*22*1";
+                      "12*322";
+                      " 123*2";
+                      "112*4*";
+                      "1*22*2";
+                      "111111"] in
```

Comment:
If this passes `ocp-indent`, why doesn't it for `connect`?

Reply:
`minesweeper` has a dedicated transformation to achieve this formatting here: https://github.com/exercism/ocaml/pull/311/files#diff-6bb359a33810bb55b6e7b4e5d31d12fcR192

Reply:
Ah, excellent. Thanks.